### PR TITLE
ch02s01: Data Science Projects

### PR DESCRIPTION
**Pull request by jramcast**
Content for https://docs.google.com/document/d/1Mijy-deKUZzIL65E4FO0gpDmznv6H-Rw4uVfllJiN0c/edit#heading=h.f3yzy8hql3ek

There is not a lot to include in this section other than how to create and manage data science projects.

Given the brevity of the content, I have embedded the exercise in the lecture.

### CODE REVIEW

File: modules/chapter2/pages/index.adoc [hunk 1]
```diff
@@ -1,3 +1,5 @@
-= Chapter 2
+= Working with Data Science Projects
 
-This is the home page of _Chapter 2_ in the *hello* quick course....
\ No newline at end of file
+Goals:
```
Comment (rsriniva) [inline modules/chapter2/pages/index.adoc:3]: Just like the chapter 1, add one or two lines about overall chapter topics and what students will learn in this chapter

File: modules/chapter2/pages/section1.adoc [hunk 2]
```diff
@@ -1,3 +1,76 @@
-= Section 1
+= Data Science Projects
 
-This is _Section 1_ of _Chapter 2_ in the *hello* quick course....
\ No newline at end of file
+In RHODS, a _data science project_ is the preferred way to organize resources when working on an AI/ML application.
+Internally, a data science project is an OpenShift project that contains specific labels to identify the project as a RHODS-specific project,
+for example:
+
+[source,yaml,subs="+quotes"]
```
Comment (rsriniva) [inline modules/chapter2/pages/section1.adoc:7]: This code block can go below the "the need for DS projects" section

File: modules/chapter2/pages/section1.adoc [hunk 3]
```diff
@@ -1,3 +1,76 @@
...
+  name: my-ai-project
+  labels:
+    kubernetes.io/metadata.name: my-ai-project
+    *modelmesh-enabled: 'true'*
+    *opendatahub.io/dashboard: 'true'*
+    ...
+----
+
+== The Need for Data Science projects
+
```
Comment (rsriniva) [inline modules/chapter2/pages/section1.adoc:21]: I wonder if this is the right place to introduce definitions from the product docs - https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science_self-managed/1.33/html-single/working_on_data_science_projects/index#working-on-data-science-projects_nb-server

That important note in the product docs is also relevant to the code block you mentioned earlier. please see how to incorporate these into the lecture

File: modules/chapter2/pages/section1.adoc [hunk 4]
```diff
@@ -1,3 +1,76 @@
...
+image::create-ds-project.png[]
+
+After you create the project, RHODS opens the project dashboard page.
+
+image::ds-project-dashboard.png[]
+
+From this page you can manage the project components and the permissions.
+
+
+=== Configure Access
```
Comment (rsriniva) [inline modules/chapter2/pages/section1.adoc:51]: Add a TODO marker - Link to the admin course section that teaches user and groups creation

File: modules/chapter2/pages/section1.adoc [hunk 5]
```diff
@@ -1,3 +1,76 @@
...
+
+image::ds-project-delete.png[]
+
+
+// Adding and deleting cluster storage to the project
+// [Jaime R] we should cover this after the creation of a workbench
+
+// == TODO: Disabled self-provisioning
+// [Trevor] Include details about how to manage a DS project when Self Provisioning is disabled on a cluster (e.g. annotations on namespaces).
+// [Jaime R] This might be more suitable for the admin-oriented quick course.
```
Comment (rsriniva) [inline modules/chapter2/pages/section1.adoc:76]: Add a TODO marker - link to the admin course section that teaches this

**Review by rsriniva — CHANGES_REQUESTED** [review]
Good one. Added some minor comments for TODO markers and product docs

File: modules/chapter2/pages/section1.adoc (hunk 3 above)
Comment (jramcast) [reply modules/chapter2/pages/section1.adoc:21]: All these definitions have been already introduced in [ch01s02](file:///home/jairamir/courses/rhods-intro/build/site/rhods-intro/1.33/chapter1/section2.html). 

I will incorporate the note, though

**Review by rsriniva — APPROVED** [review]

**Pull request merged**